src/port1.0/portmain.tcl

```diff
@@ -154,6 +154,14 @@ set egid [getegid]
 default worksymlink {[file normalize [file join $portpath work]]}
 default distpath {[file normalize [file join $portdbpath distfiles ${dist_subdir}]]}
 
+default use_xcode {[portmain::get_default_use_xcode]}
```

Comment:
This could simply be `default use_xcode {[expr {[option build.type] eq "xcode"}]}`

Reply:
Changed